### PR TITLE
Lowered encumbrance and tweaked belts

### LINKED ISSUE #45606: Balance belts
# Describe the bug
Related to #45468 i think that more items should go trough the rebalancing process, stuff like belts in general or even bandoliers have very high encoumbrance values especially if compared with most backapacks or even clothing that could carry the same items, a full tool belt has an encoubrance of 36 that's almost as much as a full hikingbackpack that has 40 on it's max capacity 

# Steps To Reproduce

simply compare various in game values 

# Expected behavior
Items like tool belts, and police belts should be if not useful items to wear at least a vaild strategy as the game stands right now you've basically no reason to wear those type fo equipment unless it's super early on and you've have no other means of carrying items. On the other hand we know that in RL such items even while full are not that uncofortable and barerly encoumbrant 

# Versions and configuration

    -- OS: Windows
    - OS Version: 10.0 1909
- Game Version: 0.E-7759-gf3a1933 [64-bit]
- Graphics Version: Tiles
- Game Language: English [en]
- Mods loaded: [
    Dark Days Ahead [dda],
    Disable NPC Needs [no_npc_food],
    Bionic Professions [package_bionic_professions]
]

### PR DESCRIPTION
**Pull request by Kelenius**
#### Summary

SUMMARY: Balance "Lowered encumbrance and tweaked belts"

#### Purpose of change

Resolves #45606

#### Describe the solution

I downloaded 0.D version, checked the stats there, and adjusted current stats based on that.

Firefighter belt:
0.D: 0 encumbrance, 1L storage + holster (2L max)
Pre-change: 4-12 encumbrance, 2L pocket belt loop
New: 2-4 encumbrance, 1L pocket + 2L pocket belt loop
Comment: I lowered the encumbrance to sane levels and gave it back some non-belt loop storage space, which seems to have been removed in an error. Its description says `Useful for keeping your bunker gear up, it also doubles as an equipment belt with a reinforced loop for holding a large tool`, so it sounds like storage space should be there.

Tool belt:
0.D: 10 encumbrance, 1L storage, +4 holsters (1L max each)
Pre-change: 2-36 encumbrance, 1L pocket, 5x 1.5L pocket belt loop or sheath
New: 2-12 encumbrance, 1L pocket, 2x 1.5L pocket belt loop or sheath, 3x 1L pocket belt loop or sheath
Comment: I lowered the encumbrance to sane levels and lowered the storage a little - most tools are small and I don't see why it needed to be increased. FWIW, I felt that 10 encumbrance was too much in 0.D and never used it.

Survivor utility belt:
0.D: 0 encumbrance, 0.5L storage + holster (1L max) + has qualities of all toolbox tools [toolbox is 2.5L]
(Obsoleted in the current version)

Survivor belt:
0.D: 1 or 3 encumbrance (refittable), 3L storage + holster (1L max)
Pre-change: 2-32 or 4-42 encumbrance (refittable), 1L pocket, 1L pocket sheath, 5x 1.5L pocket belt loop or sheath
New: 1-4 or 3-8 encumbrance, 1L pocket, 1L pocket sheath, 2x 1.5L pocket belt loop or sheath, 3x 1L pocket belt loop or sheath
Comment: Similar to tool belt.

Police duty belt:
0.D: does not exist
Pre-change: 6-15 encumbrance, 2.25L pocket belt loop
New: 2-4 encumbrance, 0.5L pocket + 2.25L pocket belt loop
Comment: Again, excessive encumbrance here, I changed it to be the same as firefighter belt. I thought that the pocket for the baton is too big (the baton is only 750 ml), but these belts can also contain tonfas, which are 2250 ml. The description says that `It has several pouches and a holder for a baton.`, so I added 500 ml of storage.

#### Describe alternatives you've considered

This is what I think the numbers should be, but I'm interested in feedback.

#### Testing

Spawned items in-game and looked at the stats.

### CODE REVIEW

**Comment by SirPendrak** [comment]
Great analysis and i would agree with most if not all changes. 

When i was changing holsters and sheaths, i usually ballanced things arround this: 
Basic backpack has 25l storage and has 25 max enc. It also has some encumbrance when empty. This gives 1 encumbrance point per 1 litter. Some better quality backpacks or some very big backpacks have better enc ratio but they come with bigger base encumbrance when they are not full. 

If an item is a holster for a weapon or something simillar that benefits much from abbility to quickdraw, then its encumbrance should be higher, probably arround 2 enc. points per litter. 
And since some items like tools dont gain much from abbility to quickdraw and just have to be in inventory, toolbelts and such shouldnt suffer as much from this rule for ballance sake.

If an item covers only one of doubled bodyparts like one leg or one hand then it should have its encumbrance doubled. See holster that has 4 points for 1 litter, but it covers one leg and allows quickdrawing pistol which is vital for this type of weapon. 

Then you can have some variation added to final encumbrance score based on quality of item (survivor stuff OP), and things like how feasible would be to carry this kind of item on that specific bodypart etc. 

But most importantly, players should be rewarded for carefully tailoring their EQ loadout. If this change will make it an obvious choice for police batton users to equip police belts, then its a good change. And small storage oriented items like belts should have at least one good selling point, be it good enc-storage ratio or one item quickdraw, because average survivor will allways have a backpack which allways can be changed to bigger one if you just need more storage. And belts and such occupy one additional layer slot. 

While youre at this, can you edit survivor belt movecost for knife sheath to 20 from 3 to match recent changes with sheaths and holsters?

**Comment by reed501** [comment]
I agree the current numbers are too high for sure (32 enc for a belt? Bruh). But I also think these new numbers may be too low. I know they're based on pre-pocket data but pockets have made belts super strong as a natural side effect. Tool belt seems pretty ok, I would've put 18-ish for max but that's not really much different in the end. My critique is for the survivor belt. Yes, it's tailored to you and hard to make but it's very durable and holds a _ton_ of stuff, which I think if you're using to its fullest extent should be more than 4 encumbrance. I've thought of belts as something to take off before combat like a backpack because I wouldn't want to do melee combat with a belt on that has two saws, a hammer, an 8" knife, a crowbar, a screwdriver set, and a 1/3 pound of spaghetti; you can store 23 pounds in that thing if you're clever. You're right though, 32 was far too much, but 4 encumbrance was probably a bit too safe to leave on for melee. If I was the one making the call I'd say 8-12 is better. The other "advanced" belts have the same basic issue to a lesser extent. Maybe I'm wrong about encumbrance values or how belts should work in this game but I believe 4 max enc will move these items into the "too good not to use" category which I think we should be avoiding.

**Comment by emptytriangle** [comment]
I was looking at halligan bars the other day and trying to figure out how you put one on a belt loop. It seems like something you'd have to be 8 feet tall to do

image

**Comment by Kelenius** [comment]
Something you're not taking into account is that most of these pockets can only hold one item. Survivor belt has 8 liters of storage in total, yes, but only one liter is "free" storage - the rest of the pockets can hold one item each. If you're putting a screwdriver set into a 1.5L pocket, you're wasting 2/3 of the storage space. Aside from saws, tools with belt clips are mostly 500 ml or less, weapons can wary, some would fill the entire pocket, some are 400 ml or smaller. Regular backpack offers 25 L of storage for 25 encumbrance; things like survivor bags or MOLLE packs have even better ratios. And the survivor belt filled with tools should take up less encumbrance than putting those tools into a backpack, or else what's the point of having a belt? 4 encumbrance is 4+ liters of space in the backpack, and hammer + hacksaw + wood saw + screwdriver set + wrench is 0.5 + 1 + 1 + 0.5 + 0.5 = 3.5 liters total. Now, survivor belt also comes with a sheath, but if I make it more encumbering, you'd be better off just tossing your stuff into the backpack and using a regular sheath. I think 4 is the sweet spot where you can choose if you want to have it or if you'd rather use a backpack.

**Comment by anothersimulacrum** [comment]
> And the survivor belt filled with tools should take up less encumbrance than putting those tools into a backpack, or else what's the point of having a belt?

Access speed.

**Comment by Kelenius** [comment]
> Access speed.

It's useful for weapons but doesn't matter at all for tools. You don't need to pull them out, they're always accessible for crafting and other purposes as long as they're in your inventory.

**Comment by anothersimulacrum** [comment]
You don't need to pull them out in game, but that doesn't mean that belts aren't going to be more encumbering than containers with slower access speeds because of it.

**Comment by emptytriangle** [comment]
> You don't need to pull them out in game, but that doesn't mean that belts aren't going to be more encumbering than containers with slower access speeds because of it.

How encumbering should they be? It's pretty easy to hit over 40 torso encumbrance with just an undershirt, shirt, kevlar vest, MOLLE pack, and a slung rifle.

What are the torso encumbrance thresholds meant to represent and are the points granular enough? Because right now putting on a tool belt for my character is enough to go from halfway into moderately encumbered to a decent bit into heavily encumbered.

I don't think the original system intended belts to have encumbrance of more than 1 or 2. It doesn't make much sense to me that someone designed this system with the intent that a tool belt might make it mechanically unviable to carry a rifle.

**Comment by anothersimulacrum** [comment]
I'm not sure it should be particularly viable to be carrying both a rifle and a full tool belt, or that's particularly reasonable behavior.
Your tool belt is for working in, not fighting. Of course you ditch that when you go out to fight.

File: data/json/items/armor/belts.json [hunk 1]
```diff
@@ -162,8 +164,10 @@
     "color": "brown",
     "covers": [ "torso" ],
     "coverage": 5,
-    "encumbrance": 6,
+    "encumbrance": 2,
+    "max_encumbrance": 4,
     "pocket_data": [
+      { "max_contains_volume": "500 ml", "max_contains_weight": "1500 g", "moves": 100 },
```
Comment (anothersimulacrum) [inline data/json/items/armor/belts.json:170]: Several pouches should probably be multiple small pockets, not a single one.

**Comment by Kelenius** [comment]
I'll get back to it soon.

**Comment by Kelenius** [comment]
Soon (tm)

Firefighter belt: after some thinking I raised max encumbrance from 4 to 5. 5 encumbrance for 3 liters of storage seems about right, considering 2 liters of it is a pocket that fits pretty big items and can be retrieved quickly, though somewhat offset by only holding one item.
Police duty belt: I increased max encumbrance from 4 to 5 for the same reason. The first pocket now only accepts small items to emulate several small pockets; I don't want to add multiple pockets to avoid interface clutter.
Survivor belt: I changed encumbrance from 1->4/3->8 to 1->6/2->12. 6 encumbrance for 8 liters is pretty good, as it's a custom-made high-tier item.
Tool belt: Left as is, at 2->12. Capacity-wise it's a survivor belt without one 1 liter knife sheath (7 liters), but it's easy to loot and isn't specifically tailored for comfort. With these numbers it's noticeably worse than regular backpack with an advantage of accepting bigger items and faster retrieval and disdvantage of inevitably wasting some space.

**Pull request merged**